plugins/modules/zos_archive.py

```diff
@@ -110,12 +110,16 @@
       - If I(dest) has missing parent directories, they will be created.
       - If I(dest) is a nonexistent USS file, it will be created.
       - Destination data set attributes can be set using I(dest_data_set).
+      - Destination data set space will be calculated based on space of
+        source data sets provided and/or found by expanding the pattern name.
+        Calculating space can impact performance, by providing space in
+        I(dest_data_set), the performance will be improved.
```

Comment:
don't remember if wording for this was discussed elsewhere, i'm thinking it could be slightly tweaked:
```
      - Destination data set space will be calculated based on space of
        source data sets provided and/or found by expanding the pattern name.
        Calculating space can impact module performance. Specifying space attributes
        in the I(dest_data_set) option will improve performance.
```

Reply:
add details for when `dest` already exists.

```
- If I(dest) is an existing file or data set and I(force=true), the existing I(dest) 
   will be deleted and recreated according to the above rules.
- If I(dest) is an existing file or data set and I(force=false) or not specified, the module
    exits with a note to the user. 
```

Please see if you can find a nicer word for `rules`, I don't like it but I couldn't come up with a nicer way of saying it.

Reply:
I'm also thinking for ordering, maybe this works best? happy to discuss other orderings too
```yml
    description:
      - The remote absolute path or data set where the archive should be
        created.
      - I(dest) can be a USS file or MVS data set name.
      - If I(dest) has missing parent directories, they will be created.
      - If I(dest) is a nonexistent USS file, it will be created.
      - If I(dest) is an existing file or data set and I(force=true), the existing I(dest)
        will be deleted and recreated according to the above rules.
      - If I(dest) is an existing file or data set and I(force=false) or not specified, the
        module exits with a note to the user. 
      - Destination data set attributes can be set using I(dest_data_set).
      - Destination data set space will be calculated based on space of
        source data sets provided and/or found by expanding the pattern name.
        Calculating space can impact performance, by providing space in
        I(dest_data_set), the performance will be improved.

```

Note - I pasted above the original contents of the `destination data set space will be calculated....` line rather than my suggested edits

Reply:
I agree with your suggestions, overall helps to clarify. And rules is good for me

Reply:
I'm rethinking the wording for this:
from 
```yml
      - If I(dest) is an existing file or data set and I(force=true),
        the existing I(dest) will be deleted and recreated according to the
        above rules.
```

to 
```yml
      - If I(dest) is an existing file or data set and I(force=true),
        the existing I(dest) will be deleted and recreated with attributes
        defined in I(dest_data_set) or computed by the module.
```

Reply:
Agree

Reply:
updated